src/cursor/cur_file.c

```diff
@@ -185,9 +185,9 @@ __curfile_search(WT_CURSOR *cursor)
 	CURSOR_API_CALL(cursor, session, search, cbt->btree);
 	WT_ERR(__cursor_checkkey(cursor));
 
-	WT_CLOCK_GET_TIME(session, start);
+	start = __wt_rdtsc(session);
 	WT_ERR(__wt_btcur_search(cbt));
-	WT_CLOCK_GET_TIME(session, stop);
+	stop = __wt_rdtsc(session);
 	 __wt_stat_read_op_histogram(session, (stop - start));
```

Comment:
Just FYI, until WT-3790 is merged these numbers are not meaningful. The TSC is only a count of clock ticks and that varies from machine to machine as to how many ticks per nanosecond.